parse_helpers.go

```diff
@@ -0,0 +1,77 @@
+package memefish
+
+import (
+	"github.com/cloudspannerecosystem/memefish/ast"
+	"github.com/cloudspannerecosystem/memefish/token"
+)
+
+func newParser(filepath, s string) *Parser {
+	return &Parser{
+		Lexer: &Lexer{
+			File: &token.File{FilePath: filepath, Buffer: s},
+		},
+	}
+}
+
+// ParseStatement parses an input string containing a statement.
+// This function won't panic but returns error if parser becomes error state.
```

Comment:
This comment seems unnecessary because users don't know `panic` is used for reporting an error on parsing.

Reply:
fixed https://github.com/cloudspannerecosystem/memefish/pull/148/commits/5cb45c07d2622712c6238e693f9c1d491c6b3d15